pages/product/product.css

```diff
@@ -0,0 +1,30 @@
+:root {
+    --border-black: #000000;
+}
+
+.filters {
+    position: sticky;
+    top: 0rem;
```

Comment:
If the value is "0", then the unit is optional. Same as the value below.

Reply:
ok will fix this 👍🏼